index.php

```diff
@@ -9,14 +9,14 @@
 $change='';
 $percentChange='';
 $theTime='';
+$theDate='';
```

Comment:
I will bring that up with the PHP Hypertext Processor committee on why they are so hell-bent on "variable declaration".  Python would be more fun here, but yea maybe let's push that to a lunch discussion.